src/java/org/apache/nutch/util/NutchJob.java

```diff
@@ -81,4 +83,26 @@ public static void cleanupAfterFailure(Path tempDir, Path lock, FileSystem fs)
     }
   }
 
+  /**
+   * Method to return job failure log message. To be used across all Jobs
+   * 
+   * @param name
+   *          Name/Type of the job
+   * @param job
+   *          Job Object for Job details
+   * @return job failure log message
+   * @throws IOException
+   *           Can occur during fetching job status
+   * @throws InterruptedException
+   *           Can occur during fetching job status
+   */
+  public static String getJobFailureLogMessage(String name, Job job)
+      throws IOException, InterruptedException {
+    if (job != null) {
+      return String.format(JOB_FAILURE_LOG_FORMAT, name, job.getJobID(),
+          job.getStatus(), job.getStatus().getFailureInfo());
```

Comment:
Really `job.getStatus()` instead of `job.getStatus.getState()` ?

The log output doesn't look readable:
```
2022-01-18 17:37:04,892 ERROR o.a.n.c.Injector [main] Injector job did not succeed, job id: job_local138302276_0001, job status: job-id : job_local138302276_0001uber-mode : falsemap-progress : 1.0reduce-progress : 0.0cleanup-progress : 1.0setup-progress : 1.0runstate : FAILEDstart-time : 0user-name : sebpriority : DEFAULTscheduling-info : NAnum-used-slots0num-reserved-slots0used-mem0reserved-mem0needed-mem0, reason: NA
```